src/constants/filters.ts

```diff
@@ -1,15 +1,21 @@
 export const keys = [
```

Comment:
I think we should add a type here to stop this from happening again. Doing so actually indicates that we have some other wrong fields here too.

```suggestion
export const keys: (keyof OracleQueryUI)[] = [
```

Adding the type shows that these keys are invalid:

"price",
 "formattedBond",
 "formattedReward",
 "assertion",

Reply:
great idea!